x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_source_list.tsx

```diff
@@ -97,12 +98,22 @@ export const AddSourceList: React.FC = () => {
     filterConfiguredSources
   ) as SourceDataItem[];
 
+  const Layout = isOrganization ? WorkplaceSearchPageTemplate : PersonalDashboardLayout;
+
   return (
-    <>
-      <ViewContentHeader title={PAGE_TITLE} description={PAGE_DESCRIPTION} />
+    <Layout
+      pageChrome={[NAV.SOURCES, NAV.ADD_SOURCE]}
+      pageViewTelemetry="add_source"
+      pageHeader={{ pageTitle: PAGE_TITLE, description: PAGE_DESCRIPTION }}
+      isLoading={dataLoading}
+    >
+      {!isOrganization && (
+        <div>
+          <ViewContentHeader title={PAGE_TITLE} description={PAGE_DESCRIPTION} />
+        </div>
+      )}
```

Comment:
There is possibly extra cleanup here on the Add Source views that could have been done (e.g. empty state, titles, etc.) in light of the new templates - but I didn't want to spend a bunch of extra time here and went with creating as few diffs as possible

Reply:
Glad you didn't go down this rabbit hole!